Slider/Assets/Scripts/Sliders/Grid/AreaSTiles/MirageSTileManager.cs

```diff
@@ -29,6 +48,34 @@ public void Save()
             UnSubscribeMirageEvents();
         }
         SaveSystem.Current.SetBool(MIRAGE_ENABLED_SAVE_STRING, mirageEnabled);
+        SaveSystem.Current.SetString(MIRAGE_TILES_SAVE_STRING, BuildMirageTilesSaveString());
+    }
+
+    private string BuildMirageTilesSaveString()
+    {
+        if(enabledMirageTiles == null) return "";
+        StringBuilder s = new();
+        if(enabledMirageTiles.Count > 2) 
+            Debug.LogWarning("more than 2 mirage tiles saved! This should not happen!");
+        for(int i = 1; i <= 7; i++)
+        {
+            MirageTileData data = null;
+            foreach(MirageTileData d in enabledMirageTiles)
+            {
+                if(d.orignalTileID == i)
+                    data = d;
+            }   
+            if(data == null)
+            {
+                s.Append("XXXX");
+            }
+            else
+            {
+                s.Append($"{data.orignalTileID}{data.buttonID}{data.x}{data.y}");
+            }
+        }
+       // print(s.ToString());
+        return s.ToString();
     }
```

Comment:
I don't think we should be encoding data into strings and saving it D: if you don't want to write stuff to serialize the MirageTileData class do you think you could at least save the data as ints instead?

i.e. 
`BuildMirageTilesSaveString(int buttonId, string parameter) => $"{MIRAGE_TILES_SAVE_STRING}_{buttonId}_{parameter}";`
`SaveSystem.Current.SetInt(BuildMirageTilesSaveString(i, "originalTileId"), data.originalTildId);`
`SaveSystem.Current.SetInt(BuildMirageTilesSaveString(i, "xPosition"), data.x);`
`SaveSystem.Current.SetInt(BuildMirageTilesSaveString(i, "yPosition"), data.y);`

Reply:
Why is having 8 ints better than 1 string?